barretenberg/cpp/src/barretenberg/stdlib/primitives/field/field.test.cpp

```diff
@@ -178,9 +178,8 @@ template <typename Builder> class stdlib_field : public testing::Test {
     {
         Builder builder = Builder();
         auto gates_before = builder.get_num_gates();
-        uint64_t expected = fidget(builder);
+        fidget(builder);
         auto gates_after = builder.get_num_gates();
-        EXPECT_EQ(builder.get_variable(builder.w_o()[gates_after - 1]), fr(expected));
         info("Number of gates added", gates_after - gates_before);
```

Comment:
The test now doesn't check that the witness corresponds to the out-of-circuit evaluation. Why was this removed?

Reply:
brought these checks back